acs_tool.py

```diff
@@ -0,0 +1,138 @@
+# uncompyle6 version 3.2.4
+# Python bytecode 2.7 (62211)
+# Decompiled from: Python 3.6.7 (default, Nov  3 2018, 21:32:46) 
+# [GCC 8.2.0]
+# Embedded file name: acs_tool.py
+# Compiled at: 2015-09-17 07:30:19
+import sys, os, os.path, json
+from struct import *
+import codecs, shutil, copy, collections
+ENTRY_POINT_OFFSET = 4
+BL2_HEADER_OFFSET = 4096
+ACS_TOOL_VERSION = 1
+acs_v1 = collections.OrderedDict()
+acs_v1['acs_magic'] = 'acs__'
+acs_v1['chip_type'] = 1
+acs_v1['version'] = 2
+acs_v1['acs_set_length'] = 8
+acs_v1['ddr_magic'] = 'ddrs_'
+acs_v1['ddr_set_version'] = 1
+acs_v1['ddr_set_length'] = 2
+acs_v1['ddr_set_addr'] = 8
+acs_v1['ddrt_magic'] = 'ddrt_'
+acs_v1['ddrt_set_version'] = 1
+acs_v1['ddrt_set_length'] = 2
+acs_v1['ddrt_set_addr'] = 8
+acs_v1['pll_magic'] = 'pll__'
+acs_v1['pll_set_version'] = 1
+acs_v1['pll_set_length'] = 2
+acs_v1['pll_set_addr'] = 8
+check_excepts = [
+ 'ddr_set_addr', 'ddrt_set_addr', 'pll_set_addr']
```

Comment:
Why a new line here ?

Reply:
You will have to ask https://github.com/rocky/python-uncompyle6 :)